edgelet/provisioning/src/provisioning.rs

```diff
@@ -249,11 +249,23 @@ where
                                         Err(Error::from(ErrorKind::Provision))
                                     },
                                     |key| {
-                                        Ok(Credentials { auth_type: AuthType::SymmetricKey(SymmetricKeyCredential{ key: Some(key.to_string()) }), source: CredentialSource::Payload })
+                                        Ok(Credentials {
+                                            auth_type: AuthType::SymmetricKey(
+                                                SymmetricKeyCredential {
+                                                    key: Some(key.to_string())
```

Comment:
Add a trailing comma `key: Some(key.to_string()),`

Same for the `source` field, and for the two in the `"hsm"` branch.

Reply:
sure